main.bicep

```
@@ -524,3 +700,10 @@ output subscriptionAcceptOwnershipState string = (subscriptionAliasEnabled && em
 
 @sys.description('The Subscription Ownership URL. Only used when creating MCA Subscriptions across tenants')
 output subscriptionAcceptOwnershipUrl string = (subscriptionAliasEnabled && empty(existingSubscriptionId) && !empty(subscriptionTenantId) && !empty(subscriptionOwnerId)) ? createSubscription.outputs.subscriptionAcceptOwnershipUrl : 'N/A'
+
+@sys.description('The resource providers that filed to register')
+output failedResourceProviders string = !empty(resourceProviders) || !empty(resourceProvidersFeatures) ? createSubscriptionResources.outputs.failedProviders : ''
```

```suggestion
output failedResourceProviders string = !empty(resourceProviders) ? createSubscriptionResources.outputs.failedProviders : ''
```

This can be simplified right as no need for an OR?